README.md

```diff
@@ -183,6 +188,20 @@ services:
         # that should be set in the context of the process
         environment:
             <env var name>: <env var value>
+
+        # (Optional) User ID for starting service as a different user. Takes
+        # precedence over "user" field.
+        user-id: <uid>
+
+        # (Optional) Username for starting service as a different user.
+        user: <username>
```

Comment:
Can we add some text in the comment that describes the behaviour if the specified user doesn't exist?

Reply:
Added sentence about that under "user" and "group" field.